### PR TITLE
Add status message about wrong layout

### LINKED ISSUE #3: 'N' and other letter buttons are not working
If your current keyboard layout other than ENGLISH, then you input is not N,WASD (example, 'т' instead 'n' in RU)

Task: Add notification about keyboard layout other than english.

- In update add checking input button for english afabet letter
- If not english -> show list StatusMessage with warning "change input layout to english"

### PR DESCRIPTION
**Pull request by mortum5**
Show status message when key exactly one rune, letter and non latin.

Resolves #3

### CODE REVIEW

**Comment by rusinikita** [comment]
Here is some UX/UI problem. Message too unnoticeable.

Screenshot 2024-05-04 at 14 13 12

Same problem with message "Code has copied to clipboard". It's unnoticeable. I placed it inside help message, tutorial and status message. There's still people who skip it.

Screenshot 2024-05-04 at 14 16 55

So, I think it's better to create our own status message on code/questions line. Something like that. And use timer command to hide it.

Screenshot 2024-05-04 at 14 28 33

Screenshot 2024-05-04 at 14 28 40

File: choose/choose.go [hunk 1]
```diff
@@ -61,6 +61,9 @@ func (c choose) Update(msg tea.Msg) (m tea.Model, cmd tea.Cmd) {
 		)
 
 	case tea.KeyMsg:
+		if play.ValidateBindings(msg) {
```
Comment (rusinikita) [inline choose/choose.go:66]: There is no problem on that screen. Let's revert it

**Review by rusinikita — CHANGES_REQUESTED** [review]
Let's do some UX improvements to notification message

File: choose/choose.go (hunk 1 above)
Comment (mortum5) [reply choose/choose.go:66]: But on this screen the user can press k/j/q/. And if the layout is incorrect, the user will not even be able to exit the application.

File: play/style.go [hunk 2]
```diff
@@ -8,6 +8,10 @@ var (
 	answerStyle         = lipgloss.NewStyle().
 				PaddingLeft(1).PaddingRight(1).
 				BorderStyle(lipgloss.NormalBorder())
+	infoMessageStyle = lipgloss.NewStyle().
```
Comment (rusinikita) [inline play/style.go]: Let's rename and use this colours. I just experimented with ascii colours. And it looks better for me

```
	warningMessageStyle = lipgloss.NewStyle().
				Foreground(lipgloss.Color("231")).
				Background(lipgloss.Color("166")).
				Padding(0, 1)
```

**Review by rusinikita — CHANGES_REQUESTED** [review]

File: play/style.go (hunk 2 above)
Comment (rusinikita) [reply play/style.go]: I use this ansi colours table to match codes

https://github.com/fidian/ansi?tab=readme-ov-file#--color-codes

File: play/bindings.go [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
-import "github.com/charmbracelet/bubbles/key"
+import (
+	"fmt"
+	"unicode"
+
+	"github.com/charmbracelet/bubbles/key"
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+const LayoutErrStatus = "Change keyboard layout to english"
```
Comment (rusinikita) [inline play/bindings.go]: Unused

File: play/bindings.go [hunk 4]
```diff
@@ -53,6 +61,22 @@ func newBindings() keyBindings {
...
 }
 
+func ValidateBindings(msg tea.KeyMsg) bool {
+	return msg.Type == tea.KeyRunes &&
+		len(msg.Runes) == 1 &&
+		unicode.IsLetter(msg.Runes[0]) &&
+		!unicode.Is(unicode.Latin, msg.Runes[0])
+}
+
+func RenderLayoutErr(msg tea.KeyMsg) string {
```
Comment (rusinikita) [inline play/bindings.go]: Let's move to style.go

**Review by rusinikita — CHANGES_REQUESTED** [review]

File: play/bindings.go (hunk 3 above)
Comment (mortum5) [reply play/bindings.go]: Done

File: play/bindings.go (hunk 4 above)
Comment (mortum5) [reply play/bindings.go]: Done

**Comment by rusinikita** [comment]
Thanks a lot!

**Comment by mortum5** [comment]
You're welcome

**Pull request merged**